test/core/end2end/tests/disappearing_server.cc

```diff
@@ -210,7 +210,9 @@ static void disappearing_server_test(grpc_end2end_test_config config) {
 
 void disappearing_server(grpc_end2end_test_config config) {
   GPR_ASSERT(config.feature_mask & FEATURE_MASK_SUPPORTS_DELAYED_CONNECTION);
+#ifndef GPR_WINDOWS /* b/148110727 for more details */
```

Comment:
It would be better to disable the combo of "proxy fixture" and "disappearing server" test in https://github.com/grpc/grpc/blob/master/test/core/end2end/generate_tests.bzl, but I don't see a straighforward way of doing this, so this is good enough.